apps/api/src/domain/models/photograph/entities/photograph.entity.ts

```diff
@@ -0,0 +1,34 @@
+import { PartialDTO } from 'apps/api/src/types/partial-dto';
+import { entityTypes } from '../../../types/entityTypes';
+import { Entity } from '../../entity';
+import PhotographDimensions from './PhotographDimensions';
+
+export class Photograph extends Entity {
+    readonly type = entityTypes.photograph;
+
+    readonly filename: string;
+
+    // TODO make this a `contributorID`
+    readonly photographer: string;
+
+    readonly dimensions: PhotographDimensions;
+
+    constructor(dto: PartialDTO<Photograph>) {
+        super(dto);
+
+        const { filename, photographer, dimensions: dimensionsDTO } = dto;
```

Comment:
Would we add other metadata here, or nest another metadata object in this model?

Reply:
This is a good question. I won't mark this as resolved, but I'll wait until the next round of modelling to work on it. It seems to me that we may want to have some kind of general `metadata` property for any of our models and there may also be specific props we need to add here.